server/app/services/applications/ProgramAdminApplicationService.java

```diff
@@ -3,27 +3,58 @@
 import static com.google.common.base.Preconditions.checkNotNull;
 
 import com.google.inject.Inject;
+import com.typesafe.config.Config;
+import java.net.URI;
+import java.util.Locale;
 import java.util.Optional;
 import models.Account;
+import models.Applicant;
 import models.Application;
 import models.ApplicationEvent;
+import models.Program;
 import repository.ApplicationEventRepository;
 import repository.ApplicationRepository;
+import services.applicant.ApplicantService;
 import services.application.ApplicationEventDetails;
 import services.application.ApplicationEventDetails.NoteEvent;
 import services.application.ApplicationEventDetails.StatusEvent;
+import services.cloud.aws.SimpleEmail;
 import services.program.ProgramDefinition;
+import services.program.StatusDefinitions.Status;
+import services.program.StatusNotFoundException;
 
 /** The service responsible for mediating a program admin's access to the Application resource. */
 public final class ProgramAdminApplicationService {
+  static final String STATUS_UPDATE_EMAIL_SUBJECT_FORMAT = "An update on your application for: %s";
+
+  private final ApplicantService applicantService;
   private final ApplicationRepository applicationRepository;
   private final ApplicationEventRepository eventRepository;
+  private final SimpleEmail emailClient;
+  private final String baseUrl;
```

Comment:
These 4 values are copied right out of ApplicantService

https://sourcegraph.com/github.com/civiform/civiform/-/blob/server/app/services/applicant/ApplicantService.java?L62